### DISCUSSION TITLE
How to use Fomantic-ui with Vue.js

### DISCUSSION THREAD

**Discussion by LuisGalicia** [fomantic/Fomantic-UI#1754] [General] [answered]
Hi there,
I would know if someone using fomantic with vue. I found a project: https://github.com/271ch/fomantic-ui-vue. But the latest version was in 2019. 
I wish to use fomantic with vue, but actually I don't know how to do that.
Thanks a lot.

**Comment by exoego** [comment, marked as answer]
Unfortunately, there is no official Vue/React/Angular support in FUI 2.x and unlikely in 2.x.
[Official support for Vue or other popular frameworks is planed for FUI 3.x](https://github.com/fomantic/Fomantic-UI/blob/develop/ROADMAP.md). 

Many components and their variants of FUI 2.x are implemented only in CSS.
So, they should work nicely with any JS framework (Vue, React, Angular...) or any Vue.js-port of FUI's JS-reqruied components, though recent new JS-required features may be missing in Vue-port,

**Reply by LuisGalicia** [reply]
I keep found an alternative solution, thanks a lot

**Comment by lubber-de** [comment]
Here is a sample jsfiddle using Vue+Fomantic+jQuery to see, it's basically working 
See https://jsfiddle.net/lubber/rv9c1Lgm/7/
